pkg/daemon/daemon.go

```diff
@@ -1448,13 +1448,20 @@ func checkV2Units(units []ign2types.Unit) error {
 // no need to check for overwrites.
 func checkV3Files(files []ign3types.File) error {
 	for _, f := range files {
+		if len(f.Append) > 0 {
+			return fmt.Errorf("found an append section when checking files. Append is not supported")
+		}
```

Comment:
I think all this errors (in other files as well) should be just warning till we fix the wrong behavior - isn't erroring out a change in behavior in itself even if it's correct

Reply:
So I thought about this a bit more. The main thing here is that even today, the files check should be failing if you were to use append. In 4.5 https://github.com/openshift/machine-config-operator/blob/release-4.5/pkg/daemon/daemon.go#L1384 if you have a MC with append, it will interpret as empty, which the file itself obviously shouldn't be. This should cause an immediate unexpected on-disk state. I can test this but I'm pretty sure this should fail. Instead of being vague about it I thought we might as well make it explicit.

Reply:
I see - so yeah, if in 4.5 we're still going degraded this makes total sense - just checking to avoid any break in behavior (even if this would be correct anyway)

Reply:
I tested just now on 4.5, if you try to install with a `append` section this will show up:

```
I0929 15:01:14.463348    2250 daemon.go:1013] Validating against pending config rendered-worker-1fbc77069595714b1690fb1060794617
E0929 15:01:14.694195    2250 daemon.go:1412] content mismatch for file /etc/motd (-want +got):
```

degrading the pool during installation